core/field_colour.js

```diff
@@ -69,6 +67,13 @@ Blockly.FieldColour.fromJson = function(options) {
  */
 Blockly.FieldColour.prototype.colours_ = null;
 
+/**
+ * By default use the global constants for colour tooltips.
```

Comment:
Clarify that null makes it use the global constants--I could see at a glance that this was null by default, but not what null meant.

Reply:
Done.